### PR TITLE
adding zoofs recipe

### PR DESCRIPTION
**Pull request by jaswinder9051998**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jaswinder9051998** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks

File: recipes/zoofs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  description: |
+    zoofs is a Python library for performing feature selection using a variety of nature-inspired wrapper algorithms.
+    The algorithms range from swarm-intelligence to physics-based to Evolutionary.
+    It's easy to use , flexible and powerful tool to reduce your feature size.
+  doc_url: https://jaswinder9051998.github.io/zoofs/
+  dev_url: https://github.com/jaswinder9051998/zoofs/tree/development
+
+extra:
+  recipe-maintainers:
+    - Jaswinder
```
Comment (BastianZim) [inline recipes/zoofs/meta.yaml:52]:
```suggestion
    - jaswinder9051998
```
This needs to be your GitHub username

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one thing then this is good to go!

**Comment by BastianZim** [comment]
@conda-forge-admin restart ci

**Review by BastianZim — APPROVED** [review]

**Pull request merged**